src/testing/vortex/java_driver/src/main/java/Main.java

```diff
@@ -0,0 +1,378 @@
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.nio.channels.WritableByteChannel;
+import java.util.HashMap;
+import java.util.Map;
+import com.tigerbeetle.AccountBatch;
+import com.tigerbeetle.Client;
+import com.tigerbeetle.IdBatch;
+import com.tigerbeetle.TransferBatch;
+import com.tigerbeetle.UInt128;
+
+/**
+ * A Vortex driver using the Java language client for TigerBeetle.
+ */
+public final class Main {
+  public static void main(String[] args) throws Exception {
+    assert args.length == 2 : "java driver requires two positional command-line arguments";
```

Comment:
I _think_ JVM disables asserts by default. So let's if and throw here? 

Or maybe not, if we care to pass the enable assertions flag when running this.